src/Employer/Employer.Web/Views/Part1/Employer/Employer.cshtml

```diff
@@ -63,7 +63,7 @@
                 </div>
 
                 <button type="submit" class="button">Save and continue</button>
-                <a asp-route="@RouteNames.Dashboard_Index_Get" class="button-link">Cancel</a>
+                <a asp-route="@RouteNames.Dashboard_Index_Get" asp-route-statusFilter="@Context.Request.Cookies[CookieNames.VacancyStatusFilter]" class="button-link">Cancel</a>

```

Comment:
Can we add this as a property on the ViewModel so we're not aware of context?

Reply:
Don't want the ViewModel to be aware of context.

Reply:
It won't be. The controller will be and just sets the enum on the VM